### ISSUE TITLE
On setup page clicking next does nothing

### ISSUE THREAD

**Issue by MitchCarter90** [CachetHQ/Cachet#2647]
So I've installed Cachet, with every Prerequisites and the entire installation of Cachet.
Now when I go test it, I get the setup page, understandable.

So on the first page, I put everything in, and click next. But it does not go to the next page.
I've tricked Chrome to show the other pages (4 of them) and none of them work, except the very last one that says **Go to Dashboard**.

 **TypeError: r.nodeName is undefined all.7086017b336c8e35b8de419383fbdc12.js:4:136**, 
**Uncaught TypeError: Cannot read property 'toLowerCase' of undefined
    at ae.fn.init.val (all.7086017….js:4)
    at all.7086017….js:23
    at all.7086017….js:23**

Does anyone have any idea?

**Comment by ArjanforGames** [comment]
Same problem here. Pulled the latest version and still no success.

**Comment by ObliviousHarmony** [comment]
There are actually two bugs in this issue.

The buttons not working is the result of e5c3ff5. When the JS was moved, the wait for jQuery initialization was removed, and so none of the selectors work anymore. You can fix this by adding $ at the very beginning of cachet.js, and then removing the () from the end of it.

The second bug is the one that shows up in the console. This is caused by the way minicolors is initialized. The current initialization should be swapped out with this:

```javascript
$('.color-code').each(function() {
        var $this = $(this);

        $this.minicolors({
            control: 'hue',
            defaultValue: $this.val() || '',
            inline: false,
            letterCase: 'lowercase',
            opacity: false,
            position: 'bottom left',
            theme: 'bootstrap'
        });
    });```

You'll have to rebuild the assets, but then it will work.

**Comment by MitchCarter90** [comment]
This might be a stupid question, but how does one rebuild these asset files.

**Comment by VincentdeWit94** [comment]
@MitchCarter90 It's simple, Be sure to have nodejs/npm installed. Then from the root dir, run `npm install` to install all the node packages. After that run `npm run prod` to rebuild the assets.

**Comment by imnvs** [comment]
Hi, I'm having this same issue. The next button is unresponsive in the status.exampledomain.com/setup page. No errors in the ..storage/logs/laravel-2017-07.25.log file or httpd error_log file. I've tried the above suggestions without success.

Pgsql, centos 7, php 7.

**Comment by jbrooksuk** [comment]
Fixed in 861f0351c5836a03cd6e52e11aa8bf4fd461711d

**Comment by ObliviousHarmony** [comment]
Thanks @jbrooksuk, I'd have submitted a pull request but I was conned into hosting our fork outside of GitHub :D
